src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/DelegatedTypes.cs

```diff
@@ -23,6 +23,11 @@ internal record DelegatedPositionParams(
     Position ProjectedPosition,
     RazorLanguageKind ProjectedKind) : IDelegatedParams;
 
+internal record DelegatedPositionAndProjectContextParams(
+    VersionedTextDocumentIdentifier HostDocument,
```

Comment:
Are you missing a commit? I was expecting this to change (and its currently identical to the record above it)